src/disasm.cpp

```diff
@@ -271,7 +271,14 @@ static void print_source_line(raw_ostream &stream, DebugLoc Loc)
 }
 #endif
 
-extern "C"
+extern "C" {
+JL_DLLEXPORT void jl_init_disasm() {
```

Comment:
They're probably not particularly expensive, they just set up a couple of data structures. Will move then to codegen_init.